pkg/pillar/scripts/device-steps.sh

```diff
@@ -21,7 +21,7 @@ AGENTS0="logmanager ledmanager nim nodeagent"
 AGENTS1="zedmanager zedrouter domainmgr downloader verifier identitymgr zedagent baseosmgr wstunnelclient"
 AGENTS="$AGENTS0 $AGENTS1"
 TPM_DEVICE_PATH="/dev/tpmrm0"
-
+XDG_RUNTIME_DIR=/var/persist/tmp
```

Comment:
Why is this in here? Let's not mix up PRs.

Reply:
There was an issue with mount path resolution. So to handle that these changes were added.

Reply:
But I think those are fixed in a different way -- so lets remove these now @adarsh-zededa

Reply:
Removed.